src/utils/helpers.ts

```diff
@@ -153,3 +153,8 @@ export function getTime() {
 
   return `${hour >= 10 ? hour : `0${hour}`}:${minute >= 10 ? minute : `0${minute}`} ${amOrPm}`;
 }
+
+export function getCurrentLanguage(guildID: string, preferredLocale: string | undefined) {
+  const prefLocale = preferredLocale ? preferredLocale : 'en_US';
+  return botCache.guildLanguages.get(guildID) ? botCache.guildLanguages.get(guildID) : prefLocale;
```

Comment:
Pretty sure this can be done in a better way. Also dont use `? :` but `||`

Reply:
I changed this, it should be better now.

Reply:
Nothing changed here?